modules/fedimint-wallet-server/src/lib.rs

```diff
@@ -325,16 +327,19 @@ impl ServerModule for Wallet {
         // TODO: We should not be panicking
         let block_count = self.get_block_count().await.expect("bitcoind rpc failed");
         let block_count_proposal = block_count.saturating_sub(self.cfg.consensus.finality_delay);
-        let consensus_block_count = self.consensus_block_count(dbtx).await;
```

Comment:
Observation: we don't care what's the consensus here. Our job is to submit our vote, and our previous vote is all that concerns us.